src/Symfony/Component/Workflow/EventListener/GuardListener.php

```diff
@@ -55,6 +56,10 @@ private function getVariables(GuardEvent $event)
     {
         $token = $this->tokenStorage->getToken();
 
+        if (null === $token) {
+            throw new InvalidTokenConfigurationException(sprintf('There are no token available for workflow %s', $event->getWorkflowName()));
```

Comment:
[...] no tokens [...]

Reply:
and the exception message should end with a dot

Reply:
Updated.